plugins/webhooks/index.js

```diff
@@ -600,6 +600,15 @@ async function createEvents(eventFactory, userFactory, pipelineFactory,
         await updateAdmins(userFactory, username, scmContext, p);
         /* eslint-enable no-await-in-loop */
 
+        const checkStartFroms = ['~tag', '~release'];
```

Comment:
I think this information should not be in this line. This should be constant as a global variable or should be in data-schema.

Reply:
Also this param name should have more information. How about `excludedStartFroms` or `excludableStartFroms`?

Reply:
Thanks! I fixed to use data-schema.
https://github.com/screwdriver-cd/data-schema/pull/338

Reply:
`EXTRA_TRIGGERS` looks good 👍